python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -102,6 +102,7 @@ def __init__(
             sync_stats=True,
             log_level=logging.INFO,
             model_dir=None,
+            model_class=None,
```

Comment:
Why do you need the model_class as an argument?
Is this argument supposed to be input/modified by users?

Reply:
oh yes, this should be removed.